### PR TITLE
Python

### LINKED ISSUE #77: [SDKTECHNO-40] Python
Context :
 - 2.7
 - 3.5
 - 3.6
 - 3.7

Recommended : 3.6

### PR DESCRIPTION
**Pull request by finalspy**
Closes #77

### CODE REVIEW

File: .github/workflows/buildOnlyModified.yml [hunk 1]
```diff
@@ -18,6 +18,14 @@ jobs:
       - name: extract branch
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Patch Docker Daemon data-root
```
Comment (PierreLeresteux) [inline .github/workflows/buildOnlyModified.yml:21]: I will add a comment with this link : 
https://github.community/t/worker-running-out-of-disk-space/18245

Just to understand, why we do that and why this code

Comment (finalspy) [reply .github/workflows/buildOnlyModified.yml]: the docker patch has been removed so this is not required anymore

File: technologies/job/python/python-3.6/test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (PierreLeresteux) [inline technologies/job/python/python-3.6/test.sh:1]: This is no use anymore ?

Comment (finalspy) [reply technologies/job/python/python-3.6/test.sh]: ok removed

**Pull request merged**